stretch_core/stretch_core/stretch_driver.py

```diff
@@ -66,6 +67,9 @@ def __init__(self):
         self.robot_mode_rwlock = RWLock()
         self.robot_mode = None
 
+        self.control_modes = ['position', 'navigation']
```

Comment:
This list is missing trajectory mode. Could you add it?

Reply:
Good catch!